src/ufoLib2/converters.py

```diff
@@ -59,7 +59,9 @@ def attrs_hook_factory(
         attribs = fields(base)
         # PEP563 postponed annotations need resolving as we check Attribute.type below
         resolve_types(base)
-        kwargs: dict[str, bool | AttributeOverride] = {}
+        kwargs: dict[str, bool | AttributeOverride] = {
+            "_cattrs_detailed_validation": conv.detailed_validation
```

Comment:
I'm still not sure if we want to allow detailed validation. Maybe yes? But that would change the exception handling I think?